{
    "type": "bundle",
    "id": "bundle--82bfbb2e-7bf0-4601-a9be-59096d77c716",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--ebc548ac-f38b-4423-ba29-5a524a604a86",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2023-05-09T00:05:20.21807Z",
            "modified": "2023-05-09T00:05:20.21807Z",
            "name": "CVE-2023-23532",
            "description": "This issue was addressed with improved checks. This issue is fixed in macOS Ventura 13.3, iOS 16.4 and iPadOS 16.4. An app may be able to break out of its sandbox",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2023-23532"
                }
            ]
        }
    ]
}